src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs

```diff
@@ -64,7 +68,7 @@ public static Parameter FromInputParameter(in InputParameter operationParameter,
             var inputType = TypeFactory.GetInputType(type);
             return new Parameter(
                 name,
-                CreateDescription(operationParameter, type, (operationParameter.Type as InputEnumType)?.AllowedValues.Select(c => c.GetValueString())),
+                CreateDescription(operationParameter, type, (operationParameter.Type as InputEnumType)?.AllowedValues.Select(c => c.GetValueString()), keepClientDefaultValue ? null : clientDefaultValue),
```

Comment:
Is there any chance that the client default value is null?

Reply:
Yes, clientDefaultValue could be null. We check this in function `CreateDescription`, and the parameter is nullable in this function.